packages/opentelemetry-node/src/NodeTracerProvider.ts

```diff
@@ -29,11 +28,7 @@ export class NodeTracerProvider extends BasicTracerProvider {
    * Constructs a new Tracer instance.
    */
   constructor(config: NodeTracerConfig = {}) {
-    if (config.scopeManager === undefined) {
-      config.scopeManager = new AsyncHooksScopeManager();
-      config.scopeManager.enable();
-    }
-    super(Object.assign({ scopeManager: config.scopeManager }, config));
+    super(Object.assign({}, config));
```

Comment:
I think `Object.assign({}, config)` is redundant now, I would use `config` directly. WDYT?